group_vars/all/general.yml

```diff
@@ -35,4 +35,4 @@ sysupgrade_preserve_custom_files:
   - /etc/wireguard/wg.key
   - /root/
 
-image_search_pattern: "*-sysupgrade.*"
\ No newline at end of file
+image_search_pattern: "*-sysupgrade.*"
```

Comment:
What is this line for? Did it maybe accidentially get into this commmit?

Reply:
Look here:
https://github.com/Freifunk-Spalter/bbb-configs/commit/49b41b4b4813315bb7cfc9587cd3ffe551f53f31

If you see the diff, philipps editor or whatever just added the newline at the end.